tests/unit/test_io_convert.py

```diff
@@ -34,28 +43,58 @@
                               FooFile,
                               get_foo_buildmanager)
 
+from zarr.storage import (DirectoryStore,
+                          TempStore,
+                          NestedDirectoryStore)
+
 
 class MixinTestCaseConvert(metaclass=ABCMeta):
     """
     Mixin class used to define the basic structure for a conversion test.
+
+    To implement a test case using this mixin we need to implement the abstract methods
+    ``setUpContainer`` and ``roundtripExportContainer``. The behavior of the mixin can
+    then be further customized via the following variables:
+
+    :ivar IGNORE_NAME: Bool parameter passed to assertContainerEqual (False)
```

Comment:
note that these variables are actually class variables, not instance variables, and I think `:ivar name:` will not work here. Also, as a class variable, if one test changes the class, I think it will change for all tests that use the class downstream. It seems like these variables are modified only in the class definition, so that should be OK

Reply:
I changed the docs for the mixin classes to document the class members directly and also added a note to the module to clarify that the Mixins should always be used through inheritance.